integration/qbft/scenarios/round_change.go

```
@@ -103,14 +103,14 @@ func roundChangeInstanceValidator(consensusData []byte, operatorID spectypes.Ope
 			Data: consensusData,
 		}).Encode()
 		if err != nil {
-			return fmt.Errorf("encode prepare data: %w", err)
+			panic(err)
```

why panic instead of return err?

---

same in line 113